### DISCUSSION TITLE
How to get some of the new achievements?

### DISCUSSION THREAD

**Discussion by imanoar** [tarikmanoar/tarikmanoar#2] [Q&A] [answered]
**YOLO** - tried to push to a private repo, but did not achieve it when I checked this morning still.
Pull Shark - Have merged at least 2 PRs in a private repo (forked from friend), but did not achieve it.
Galaxy brain - Do I have to be the one to create two discussions? and mark them as answers?

Seems a bit unclear and random on how to achieve the new achievements.

Please help if anyone has a clue...

**Comment by imanoar** [comment]
My understanding is that somebody needs to be assigned as a reviewer to the PR (top right on desktop) and the PR then be merged without that person (or anybody else) doing a review.

**Comment by tarikmanoar** [comment, marked as answer]
I got the badge without doing any of the things mentioned above, I simply made commits on another branch to add a feature and merged the branch into main. You can see my actual PR https://github.com/ajmeese7/8bit/pull/1, as evidenced by the badge information on my user profile.

Accept if this helps, I'm looking forwards to getting the discussion award too smile
